### PR TITLE
Update en-US.json

### PR DESCRIPTION
**Pull request by ChgoChad**
I completed the **`{ext-letsencrypt}`** section with all of the proper descriptions based on Plesk online documentation.

### CODE REVIEW

File: en-US.json [hunk 1]
```diff
@@ -12,11 +12,28 @@
     "timeout": "The time limit (in seconds) for sending requests to issue an SSL/TLS certificate to the CA server. Helps avoid long wait times if there are issues with the connection or the CA server itself. Accepts the following values: null (default HTTP timeout values), positive integers (we recommend 30 seconds), and 0 (no timeout, not recommended)."
   },
   "ext-letsencrypt": {
-    "renew-before-expiration" : "Number of days before certificate renewal.",
-    "log-requests" : "Log requests to the ACME server in the Plesk log.",
-    "send-notifications-interval" : "Determines how often the Let’s Encrypt extension can send you notifications.",
+    "renew-before-expiration": "Number of days before certificate renewal.",
```
Comment (Eevee-chan) [inline en-US.json:15]: The number of days before the certificate renewal.

File: en-US.json [hunk 2]
```diff
@@ -12,11 +12,28 @@
...
+    "log-requests": "Log requests to the ACME server in the Plesk log.",
+    "send-notifications-interval": "Determines how often the Let’s Encrypt extension can send you notifications.",
     "key-algorithm": "OpenSSL key type. Can be RSA or ECDSA.",
-    "ecdsa-curve-name": "Curve name used in Elliptic curve cryptography (ECC) for public/private key operations. The setting accepts the following values: prime256v1 (NIST P-256) and secp384r1 (NIST P-384)."
+    "ecdsa-curve-name": "Curve name used in Elliptic curve cryptography (ECC) for public/private key operations. The setting accepts the following values: prime256v1 (NIST P-256) and secp384r1 (NIST P-384).",
```
Comment (Eevee-chan) [inline en-US.json:19]: The curve name

File: en-US.json [hunk 3]
```diff
@@ -12,11 +12,28 @@
...
+    "secure-new-domain": "boolean  Set the default state of the “Secure the domain with Let’s Encrypt” checkbox shown when creating a new subscription, domain, or subdomain.",
+    "disable-cleanup": "boolean  Disable the cleanup of token files after a domain dispute is resolved.",
+    "verify": "The path to the trusted CA Root Certificates bundle.",
+    "config-dir": "The path where certificates for third-party integration are stored.",
+    "check-availability-delay": "*The wait time in seconds between attempts to check if a domain is accessible via HTTP.",
+    "check-availability-max-attempts": "*The maximum number of attempts to check if a domain is accessible via HTTP.",
+    "check-availability-timeout": "*Timeout in seconds for checking if a domain is accessible via HTTP. If within the time defined by check-availability-timeout, no response code is received, the domain is considered to be unavailable.",
```
Comment (Eevee-chan) [inline en-US.json:30]: The timeout

**Comment by ayamshanov** [comment]
@ChgoChad, thanks for your contribution!

**Pull request merged**